bindings/container.cpp

```diff
@@ -13,28 +13,44 @@ Container::Container(QObject *parent) : QObject(parent)
 
 }
 
-void Container::installApp(QString path) {
-    if(!path.startsWith(":/")) path = QUrl(path).toLocalFile();
+Dna* loadDnaFile(QString path) {
+    if(!path.startsWith(":/"))
```

Comment:
is this about mac ":" separator for paths?  why the colon?

Reply:
Oh, no that is Qt's resource system. I've put DNAs as resources in this project so that HoloSqape starts off with something installed. Made it easier to get a running app on my Android phone where I don't have the system tray, and this was easier than writing a special UI for that case..

So: any path starting with a colon is understood by all Qt functions as pointing into the compiled in resources..